crates/matrix-sdk/src/room/mod.rs

```diff
@@ -2957,6 +2962,54 @@ impl Room {
             .await?;
         Ok(())
     }
+
+    /// Remove verification requirements for the given users and
+    /// resend messages that failed to send because their identities were no
+    /// longer verified (in response to
+    /// `SessionRecipientCollectionError::VerifiedUserChangedIdentity`)
+    #[cfg(feature = "e2e-encryption")]
+    pub async fn withdraw_verification_and_resend(
```

Comment:
Should these two methods live in the `Encryption` API object instead? They don't make any usage of the room, except for getting the send queue, which I argued below isn't the right place to do it.

Reply:
Also these two functions would need individual tests please 😁

Reply:
> these two functions would need individual tests

I reckon that if we split them up from the resending they would just proxy to the crypto crate where they're already tested.